content/docs/command-reference/commit.md

```diff
@@ -21,7 +21,7 @@ The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/dag) is in development/experimentation; to
 force-update the `dvc.lock` or `.dvc` files without reproducing stages or
-pipelines; or to mark existing files/dirs as stage <abbr>outputs</abbr>. These
+pipelines; or when modifying stage <abbr>outputs</abbr> without DVC. These
```

Comment:
All this paragraph is too complicated and sometimes wrong: e.g. `when data already
tracked by DVC changes` contradicts to the last point. 

it's hard to get the idea behind the last point (not sure the previous version is better)

Reply:
> `when data already changes`... when data already

That is the previous version 🙂 I didn't change that part.

I agree this sentence is too complex. The small change here is a small simplification but I can def. work on it more. Keeping in mind this is the cmd ref though so some complexity should be OK.

Reply:
> when data already changes

BTW, I removed that bit.